### PR TITLE
[Event Hubs Client] Track Two: Fifth Preview (Remove Embedded Track One)

### PR DESCRIPTION
**Pull request by jsquire**
# Summary

The focus of these changes is to remove the track one compatibility layer, including the embedded client and associated support constructs.

# Last Upstream Rebase

Monday, October 21, 4:26pm (EDT)

# Related and Follow-Up Issues

- [Remove embedded track one code and dependencies](https://github.com/Azure/azure-sdk-for-net/issues/8221) (#8221)

### CODE REVIEW

File: sdk/eventhub/Azure.Messaging.EventHubs/src/Azure.Messaging.EventHubs.csproj [hunk 1]
```diff
@@ -49,34 +49,11 @@
 
   <!-- Import Azure.Core shared source -->
   <ItemGroup>
-    <Compile Include="$(AzureCoreSharedSources)Argument.cs" Link="SharedSource\Argument.cs"/>
-    <Compile Include="$(AzureCoreSharedSources)ClientDiagnostics.cs" Link="SharedSource\ClientDiagnostics.cs"/>
-    <Compile Include="$(AzureCoreSharedSources)DiagnosticScope.cs" Link="SharedSource\DiagnosticScope.cs"/>
+    <Compile Include="$(AzureCoreSharedSources)Argument.cs" Link="SharedSource\Argument.cs" />
```
Comment (jsquire) [inline sdk/eventhub/Azure.Messaging.EventHubs/src/Azure.Messaging.EventHubs.csproj]: Because.... formatter?

**Review by kinelski — APPROVED** [review]
LGTM

**Pull request merged**